man/systemd.mount.xml

```diff
@@ -241,6 +241,14 @@
       </varlistentry>
 
       <varlistentry>
+        <term><option>x-systemd.mount-timeout=</option></term>
+
+        <listitem><para>Configure how long systemd should wait for the
+        mount command to finish. See <varname>MountSec=</varname> below
+        for details.</para>
```

Comment:
I think you should duplicate the description of caveats from `x-systemd.device-timeout` here. They all apply equally well.